File: SquareCheck.java
class SquareCheck {
 
	static ArrayList perfectSquares;
 	static
 	{
  		perfectSquares = new ArrayList<>();
 
  		perfectSquares.add('0');
  		perfectSquares.add('1');
  		perfectSquares.add('4');
  		perfectSquares.add('9');
 	}
 
 	public static void main(String[] args) 
 	{
		long square = 0L;
  
  		for(int i = 99; i > 31; -- i)
  		{
   			square = i * i;
   			if(areThemDigitsAlsoSquares(Long.toString(square).toCharArray()))
   			{
    			    System.out.println(square);
    			    break;
   			}
  		}
 	}
 
 	private static boolean areThemDigitsAlsoSquares(char[] array) 
 	{
		for(char c: array)
   			if(!perfectSquares.contains(c))
    			    return false;
 
  		return true;
 	}
}

File: SquareCheck.py
    def areThemDigitsAlsoSquares(string):
    	length = len(string);
     
     	for i in range(0, length):
    		character = string[i];
     
     		if not(character == '0' or character == '1' or character == '4' or character == '9'):
    			return False;
     	return True;
     
    for i in range(99, 31, -1):
    	square = i * i;
     
     	if areThemDigitsAlsoSquares(str(square)):
    		print(square);
    		break;
